protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1698,10 +1698,17 @@ message ListTransfersResponse {
   TransferConnection transfers = 1;
 }
 
+message TransferNode {
+  // Data relating to a transfer that has been made.
+  vega.events.v1.Transfer transfer = 1;
+  // Fees paid for this transfer.
+  repeated vega.events.v1.TransferFees fees = 2;
+}
+
 // Transfers data with the corresponding cursor.
 message TransferEdge {
   // Data relating to a transfer that has been made.
-  vega.events.v1.Transfer node = 1;
+  TransferNode node = 1;
```

Comment:
I think this makes it a breaking change, which annoyingly means we might have to have a whole new API unless theres is another way.

Reply:
The motivation for this change is that the current API is incomplete (some would even argue defective/broken), so a breaking change to a broken part of the API arguably isn't a bad thing. There's one alternative that _might_ reduce the number of new types here, and that would be to overload the transfers event type to contain a slice of fee payments, but that causes issues further down the line (and elsewhere in the code)